screens/auth/WelcomeScreen.js

```diff
@@ -6,11 +6,8 @@ import {
     View,
     StatusBar
 } from 'react-native';
-import { TransText } from 'react-native-translation';
-import LinearGradient from 'react-native-linear-gradient';
-import LanguageFlags from './welcome/LanguageFlags';
-import Slides from './welcome/Slides';
-import { Colors, Body, Title } from '../components';
+import { LanguageFlags, Slides } from './_components';
```

Comment:
./_ components?

Reply:
thats the naming convention I am following from start for components only used by that screen.

I will change name to authComponents and userComponents instead of _components.. as you suggested below